builders/pallets-precompiles/precompiles/xcm-utils.md

```diff
@@ -0,0 +1,82 @@
+---
+title: XCM-Utils预编译合约
+description: 通过Moonbeam的预编译XCM-Utils合约，了解可供智能合约开发者使用的各类XCM相关实用性函数。
+keywords: solidity, 以太坊, xcm, utils, moonbeam, 预编译, 合约
+---
+
+# 与XCM-Utils预编译交互
+
+![Precomiled XCM-Utils Banner](/images/builders/pallets-precompiles/precompiles/xcm-utils/xcm-utils-banner.png)
+
+## 概览 {: #xcmutils-precompile}
+
+XCM-utils预编译合约为开发者提供了直接在EVM中与XCM相关的实用性函数。这允许开发者能够更轻松地与其他XCM相关的预编译进行交易和交互。
+
+与其他[预编译合约](/builders/pallets-precompiles/precompiles/){target=_blank}类似，XCM-utils预编译位于以下地址：
+
+=== "Moonbeam"
+     ```
+     {{networks.moonbeam.precompiles.xcm_utils }}
+     ```
+
+=== "Moonriver"
+     ```
+     {{networks.moonriver.precompiles.xcm_utils }}
+     ```
+
+=== "Moonbase Alpha"
+     ```
+     {{networks.moonbase.precompiles.xcm_utils}}
+     ```
+
+--8<-- 'text/precompiles/security.md'
```

Comment:
I don't see this line in PR#466.

Reply:
It was added in a separate PR but I knew I needed to add it here once this page was added, because we had already added it to all of the other precompile pages in the CN repo. You can take a look at the `master` branch on the English repo for this page and see that it is there: https://github.com/PureStake/moonbeam-docs/blame/master/builders/pallets-precompiles/precompiles/xcm-utils.md#L32

Reply:
Thanks for clarifying it!